src/content/faq/General.md

```diff
@@ -3,25 +3,80 @@ title: Frequently Asked Questions (FAQ)
+
 ## Is GraphQL a database language like SQL?
 
+No, but this is a common misconception.
+
+GraphQL is a specification typically used for remote client-server communications. Unlike SQL, GraphQL is agnostic to the data source(s) used to retrieve data and persist changes. Accessing and manipulating data is performed with arbitrary functions called [resolvers](/learn/execution/). GraphQL coordinates and aggregates the data from these resolver functions, then returns the result to the client. Generally, these resolver functions should delegate to a [business logic layer](/learn/thinking-in-graphs/#business-logic-layer) responsible for communicating with the various underlying data sources. These data sources could be remote APIs, databases, [local cache](/learn/caching/), and nearly anything else your programming language can access.
+
+For more information on how to get GraphQL to interact with your database, check out our [documentation on resolvers](/learn/execution/#root-fields-resolvers).
+
 ## Does GraphQL replace REST?
 
+No, not necessarily. They both handle APIs and can [serve similar purposes](/learn/thinking-in-graphs/#business-logic-layer) from a business perspective. GraphQL is often considered an alternative to REST, but it’s not a definitive replacement.
+
+GraphQL and REST can actually co-exist in your stack. For example, you can abstract REST APIs behind a [GraphQL server](https://www.howtographql.com/advanced/1-server/). This can be done by masking your REST endpoint into a GraphQL endpoint using [root resolvers](/learn/execution/#root-fields-resolvers). 
+
+For an opinionated perspective on how GraphQL compares to REST, check out [How To GraphQL](https://www.howtographql.com/basics/1-graphql-is-the-better-rest/).
+
 ## How can I learn GraphQL?
 
+There are many resources available to help you learn GraphQL, including this website. In [our documentation](/learn/), you’ll find a series of articles that explain essential GraphQL concepts and how they work. Our [Community page](/community) is full of resources to reference and groups to join.
+
+For more practical guides, visit the [How to GraphQL](https://www.howtographql.com/) fullstack tutorial website. We also have a free online course with edX, [Exploring GraphQL: A Query Language for APIs](https://www.edx.org/course/exploring-graphql-a-query-language-for-apis). 
+
 ## Is GraphQL frontend or backend?
 
-## Does GraphQL replace Redux or other state management libraries?
+Neither. Instead, it’s [how you can exchange information between the two](https://www.howtographql.com/basics/3-big-picture/). Both the client and the server should understand and communicate through the GraphQL specification. 
+
+That said, GraphQL describes a specific way to [execute a query](/learn/execution/), which typically happens on the backend.
```

Comment:
I'm a bit uncomfortable with this; I feel like the execution (on the backend) is a significant component of GraphQL. I wonder if it could be more of a "both" rather than "neither"? Maybe something like:

> GraphQL specifies how you can exchange information between client and server, including how the server can indicate what data and operations are available [link to introspection], how the client should format requests [link to query language], how the server should execute these queries [link to execution], and what the client will receive in response [link to "response is same shape as query"].

Reply:
Absolutely - I really like that, so I'll update!

The only thing I'm hung up on is that I think FAQs are most effective when they answer the question right away... could it make sense to literally say like `It interacts with both, actually.` (or something better lol) at the beginning?

Reply:
I think just starting with `Both.` instead of `Neither.` might be sufficient.